Services/CO.CDP.DataSharing.WebApi/DataService/DataService.cs

```diff
@@ -10,21 +10,81 @@ namespace CO.CDP.DataSharing.WebApi.DataService;
 
 public class DataService(IShareCodeRepository shareCodeRepository, IConnectedEntityRepository connectedEntityRepository) : IDataService
 {
+    private const string DeclarationsSectionName = "Declarations";
+
     public async Task<SharedSupplierInformation> GetSharedSupplierInformationAsync(string shareCode)
     {
         var sharedConsent = await shareCodeRepository.GetByShareCode(shareCode)
                             ?? throw new ShareCodeNotFoundException(Constants.ShareCodeNotFoundExceptionMessage);
-
+        var allFormSectionsExceptDeclaractions = sharedConsent.AnswerSets.Where(a => a.Section.Title != DeclarationsSectionName);
```

Comment:
Can improve with below code, this is what my earlier comment was pointing too.
```
var allFormSectionsExceptDeclaractions = sharedConsent.AnswerSets.Where(a =>
    a.Section.Type != OrganisationInformation.Persistence.Forms.FormSectionType.Declaration);
```

Reply:
Ah yes ok. I wasn't aware of that constant. I've updated the code as you've suggested.